### ISSUE TITLE
Upgrade from C107 -> C108 on Android ARM x64 for 1.47.x - Nightly (Checking C108)

### ISSUE THREAD

**Issue by kjozwiak** [brave/brave-browser#26713]
As per process, QA runs through the following cases to ensure that the major chromium bump that's about to get merged into `master` doesn't regress the `Nightly` channel. Once the major chromium bump is merged into `master`, QA will run through a full manual pass on `Windows` & `Android`.

### Startup & Components

- [x] Verify that Brave is only contacting `*.brave.com` endpoints on first launch using either `Charles Proxy`, `Fiddler`, `Wireshark` or `LittleSnitch` (or a similar application)
  - [x] Verify that opening a NTP doesn't trigger any outbound connections related to widgets without user interaction
- [x] Restart the browser, load `brave://components`, wait for 8 mins and verify that no component shows any errors

**Note:** Always double check `brave://components` to make sure there's no errors/missing version numbers. We basically want to make sure that components are working as they regressed in C98 when we first merged into `master`.

### Upgrade

- [x] Make sure that data from the last version appears in the new version OK
- [x] Ensure that `brave://version` lists the expected Brave & Chromium versions
- [x] With data from the last version, verify that:
  - [x] Bookmarks on the bookmark toolbar and bookmark folders can be opened
  - [x] Cookies are preserved
  - [x] Installed extensions are retained and work correctly
  - [x] Opened tabs can be reloaded
  - [x] Stored passwords are preserved
  - [x] Sync chain created in previous version is retained
  - [x] Social media-blocking buttons changes are retained
  - [x] Rewards
    - [x] Wallet balance is retained
    - [x] Auto-contribute list is retained
    - [x] Both Tips and Monthly Contributions are retained
    - [x] Wallet panel transactions list is retained
    - [x] Changes to rewards settings are retained
    - [x] Ensure that Auto Contribute is not being enabled when upgrading to a new version if AC was disabled
  - [x] Ads
    - [x] Both `Estimated pending rewards` & `Ad notifications received this month` are retained
    - [x] Changes to ads settings are retained
    - [x] Ensure that ads are not being enabled when upgrading to a new version if they were disabled
    - [x] Ensure that ads are not disabled when upgrading to a new version if they were enabled

**Comment by Uni-verse** [comment]
Verified on Samsung Galaxy S21 5G using the following build(s):

```
Brave	1.47.65 Chromium: 108.0.5359.40 (Official Build) canary (32-bit) 
Revision	280b5fcaab3e877562b06cfaf2eb51121e13c3b9-refs/branch-heads/5359@{#689}
OS	Android 12; Build/SP1A.210812.016
```

Example | Example | Example
-|-|-
![screenshot-1668451246772](https://user-images.githubusercontent.com/17885425/201741315-91177bb9-7271-4553-a70f-ab2bc3887f92.png) | ![screenshot-1668451270262](https://user-images.githubusercontent.com/17885425/201741319-0ffe46d5-870c-4e1e-a4ce-a38d459c6d6b.png) | ![Screen Shot 2022-11-14 at 1 46 08 PM](https://user-images.githubusercontent.com/17885425/201741577-b9bba93f-e243-4aeb-874d-74e4b5beb812.png)
